cms/djangoapps/contentstore/views/course.py

```diff
@@ -496,6 +547,8 @@ def course_index(request, course_key):
         except (ItemNotFoundError, CourseActionStateItemNotFoundError):
             current_action = None
 
+        ora1_deprecation_info = _deprecation_info(course_module, DEPRECATED_BLOCK_TYPES)
```

Comment:
Again, is it possible to talk about this as a general deprecation mechanism where the ORA1 types are just listed in configuration, rather than having ora1 specifics in code?